src/app/_components/BlockList/LayoutA/__tests__/useBlockList.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { renderHook, waitFor } from '@testing-library/react';
+
+import '../mocks';
+import { useBlockList } from '../useBlockList';
+
+const LAST_BURN_BLOCK_STX_BLOCKS_COUNT = 50;
+
+jest.mock('../useBlockListWebSocket', () => ({
+  useBlockListWebSocket: jest.fn().mockImplementation(() => ({
+    latestUIBlocks: [],
+    latestBlocksCount: 0,
+    clearLatestBlocks: jest.fn(),
+    latestBlock: undefined,
+  })),
+}));
+
+jest.mock('../useInitialBlockList', () => ({
+  useInitialBlockList: jest.fn().mockImplementation(() => ({
+    lastBurnBlock: {
```

Comment:
This is just a generic question. I'm not quite understanding why we need initial and second to last data. Could you explain it to me?

Reply:
Interesting. ok